pkg/asset/manifests/gcp/cloudproviderconfig.go

```diff
@@ -36,8 +36,9 @@ func CloudProviderConfig(infraID, projectID, subnet, networkProjectID string) (s
 			Regional:  true,
 			Multizone: true,
 
-			// To make sure k8s cloud provide has tags for firewal for load balancer.
-			NodeTags:                     []string{fmt.Sprintf("%s-master", infraID), fmt.Sprintf("%s-worker", infraID)},
+			// To make sure k8s cloud provide has tags for firewall for load balancer.
+			// The CAPI gcp provider uses the node tag "control-plane" for master nodes.
+			NodeTags:                     []string{fmt.Sprintf("%s-master", infraID), fmt.Sprintf("%s-control-plane", infraID), fmt.Sprintf("%s-worker", infraID)},
```

Comment:
Why are we adding the worker tag here too?

Reply:
The worker node tag wasn't added with this PR, it was existing. For backwards compatibility I don't think we can/should remove it. If we do, it would have to be in another PR.